beyondtrust-privileged-remote-access/connector.py

```diff
@@ -0,0 +1,29 @@
+""" Copyright start
```

Comment:
Change the copy right year, do the same in all files

Reply:
Done